source/ClientDebug/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should not be a part of ClientDebug. Remove these packages from ClientDebug.